### ISSUE TITLE
About training 

### ISSUE THREAD

**Issue by YangChen1234567** [mit-han-lab/bevfusion#243]
Hi~
Thanks for your nice work.... But when reproducing the results, i met some problems.

I used the training instructions as below to train fusion model(diden't load pretrained lidar model), 
`torchpack dist-run -np 1 python tools/train.py configs/nuscenes/det/transfusion/secfpn/camera+lidar/swint_v0p075/convfuser.yaml`
but my val mAP is always zero, like this:(scalars from tensorboard, trained after 5 epochs)
![c257963096188f6a03b37889e9975eb](https://user-images.githubusercontent.com/44695720/203894466-caacb048-41f6-4836-86be-782104250e8e.png)

Is it normal? Is there any details when training that i dont'know??  
This is the traing loss:
![ab29bcd971c3fd74cfc927f89e2a39f](https://user-images.githubusercontent.com/44695720/203894526-284df426-19b3-489b-b006-e47649b19a76.png)

**Comment by YangChen1234567** [comment]
and my batchsize is 2 (2 GPUS, samples_per_gpu = 1)

**Comment by kentang-mit** [comment]
Hi @YangChen1234567,

Sorry for missing that issue. I believe you should use the training command for the fusion model provided by us in the current main branch.

Best,
Haotian

**Comment by kentang-mit** [comment]
Closed due to inactivity. Please feel free to reopen if you feel it necessary.
